src/mapeo-project.js

```diff
@@ -206,6 +208,13 @@ export class MapeoProject {
       projectId: this.#projectId,
     })
 
+    this.#iconServer = createIconServer({
+      logger: true,
+      prefix: '/icon/',
+      coreManager: this.#coreManager,
+      iconDataType: this.#dataTypes.icon,
+    })
```

Comment:
are we creating an icon server for each project? my initial thought is that this would be handled similarly to how we're planning to do blobs i.e. one shared server with routes prefixed by the kind of server (e.g. `/blob/` or `/icon/`) and the routes accepting a `projectId` param to work across all projects

Reply:
This first commit just starts to put everything into place by using the blob-server implementation as a reference. 
For now the blob-server is instantiated inside `mapeo-project` so I was assuming it was handled per project. 
But what you say - I think - makes sense: only have one server per MapeoManager and use `:projectId` as a param.

Reply:
> For now the blob-server is instantiated inside mapeo-project so I was assuming it was handled per project.

the `BlobApi` is instantiated in each project, but the server isn't. We have yet to actually integrate any server into mapeo manager/project

Reply:
Yeah, I think #291 is made to implement that

Reply:
mhm, from what I see in `mapeo-project.js` in main, it seems that the server is instantiated on every project (see [here](https://github.com/digidem/mapeo-core-next/blob/9d465d388e76633fe54c0df46d999cb6142f4948/src/mapeo-project.js#L202)). But maybe I'm missing smth and `createBlobServer` acts as a singleton that instantiates the server if it doesn't exists yet??

Reply:
ah no, you're right. totally missed that and was only fixated on the blob api instantiation, my bad 😅

Reply:
> ah no, you're right. totally missed that and was only fixated on the blob api instantiation, my bad 😅

I mean, it makes sense to have only one server per `MapeoManager` and not per `MapeoProject`. More so if the `projectId` is a param to the server; if not, it kinda feels redundant to pass the `projectId` ? (since we're always talking about the same instance of `MapeoProject` ).